fedn/fedn/network/controller/control.py

```diff
@@ -98,118 +114,126 @@ def session(self, config):
         # Execute the rounds in this session
         for round in range(1, int(config['rounds'] + 1)):
             # Increment the round number
-
             if last_round:
                 current_round = last_round + round
             else:
                 current_round = round
 
             try:
-                _, round_data = self.round(config, current_round)
+                _, round_data = self.round(config, str(current_round))
```

Comment:
how about using model_id for the seed model for the round as round_id? Instead of a str(int).

Reply:
I think it should be better to keep these ides separate.